css/style.css

```diff
@@ -164,6 +165,7 @@ header {
 
 .login-button:hover {
   box-shadow: inset 130px 0 0 0 #0da1dc;
+  transition: 0.5s ease-in-out;

```

Comment:
If you want to code like a professional, there is no need to code like this:
```diff
- transition: 0.3s ease-in-out;
+ transition: .3s ease-in-out;
```
Overall looking Great.